FreeRTOS/iot-reference-nxp-rt1060
# Update to LTS 3.0 libraires

Description
-----------
* Update to LTS3.0 candidate version library
* Remove signing_key.pem and signing_pub_key.c 
* Add configTASK_NOTIFICATION_ARRAY_ENTRIES in each demo for OTA
* Exclude FreeRTOS-Kernel/examples in build

LTS3.0 libraries
| Library | Version |
|---------------------------| ---- |
| FreeRTOS Kernel | 11.1.0 |
| coreJSON | 3.3.0 |
| coreMQTT | 2.3.0 |
| corePKCS11 | 3.6.1 |
| backoffAlgorithm | 1.4.1 |
| device defender | 1.4.0 |
| device shadow | 1.3.0 |
| MQTT streams | 1.1.0 |
| Jobs | 1.5.1 |

Test Steps
-----------
Verify with demo and library integration test.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
